src/utils.ts

```diff
@@ -59,12 +60,23 @@ export function getPartialMatcher(patterns: string[], options?: PartialMatcherOp
 // #endregion
 
 // #region splitPattern
+// Make the options a global constant to reduce GC work
+const SplitPatternOptions = { parts: true }
```

Comment:
sounds redundant considering objects are used in multiple places and splitting patterns aren't done in performance critical areas. also should follow camelCase and not PascalCase